tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/character_preferences/skyrat/species_features.tsx

```diff
@@ -20,61 +20,63 @@ export const feature_mcolor3: Feature<string> = {
+  description:
+    "Appears when your character is examined (but only if they're identifiable - try a gas mask).",
   component: FeatureTextInput,
 };
 
 export const silicon_flavor_text: Feature<string> = {
   name: 'Flavor Text (Silicon)',
-  description: 'Describe your cyborg/AI shell!',
+  description: "Only appears if you're playing as a borg/AI.",
   component: FeatureTextInput,
 };
 
 export const ooc_notes: Feature<string> = {
   name: 'OOC Notes',
-  description: 'Talk about your character OOCly!',
+  description:
+    'Primarily for things like ERP, general, antag/whatever preferences.',
   component: FeatureTextInput,
 };
 
 export const custom_species: Feature<string> = {
   name: 'Custom Species Name',
-  description:
-    "Want to have a fancy species name? Put it here, or leave it blank if you want to use your species' default name.",
   component: FeatureShortTextInput,
 };
 
 export const custom_species_lore: Feature<string> = {
   name: 'Custom Species Lore',
-  description:
-    "Add some lore for your species! Won't show up if there's no custom species.",
+  description: "Won't show up if there's no custom species.",
   component: FeatureTextInput,
 };
-// SKYRAT EDIT ADDITION BEGIN -- RECORDS REJUVINATION
 export const general_record: Feature<string> = {
   name: 'Records - General',
-  description:
-    'Your general records! These are records that are for general viewing-things like employment, qualifications, etc. By default, anyone with a HUD/records access can view these.',
+  description: 'Viewable by anyone with any records access.',
   component: FeatureTextInput,
 };
 
 export const security_record: Feature<string> = {
   name: 'Records - Security',
-  description:
-    'Your security records! These are records for criminal records, arrest history, things like that. Sec officers can view these.',
   component: FeatureTextInput,
 };
 
 export const medical_record: Feature<string> = {
   name: 'Records - Medical',
-  description:
-    'Your medical records! These are records for things like medical history, prescriptions, DNR orders, etc. Medical staff can view these.',
   component: FeatureTextInput,
 };
 
 export const exploitable_info: Feature<string> = {
   name: 'Records - Exploitable',
   description:
-    "This section is for information antagonists can use, IN CHARACTER AND OUT. If you are willing to be disrupted by antagonists MORE than the average player (this is usually a very fun experience, if you're into that kind of roleplay), put it here! Also for things antagonists can use against your character.",
+    'Can be IC or OOC. Viewable by certain antagonists/OPFOR users, as well as ghosts. Generally contains \
+  things like weaknesses, strengths, important background, trigger words, etc. It ALSO may contain things like \
+  antagonist preferences, e.g. if you want to be antagonized, by whom, with what, etc. DO NOT PUT SEXUAL THINGS IN HERE! THAT IS FOR OOC NOTES!',
```

Comment:
```suggestion
    'Can be IC or OOC. Viewable by certain antagonists/OPFOR users, as well as ghosts. Generally contains \
  things like weaknesses, strengths, important background, trigger words, etc. It ALSO may contain things like \
  antagonist preferences, e.g. if you want to be antagonized, by whom, with what, etc.',
```
For now let's just leave it out.